packages/react/src/components/ListBox/ListBoxSelection.js

```diff
@@ -49,17 +48,17 @@ const ListBoxSelection = ({
   };
   const description = selectionCount ? t('clear.all') : t('clear.selection');
   return (
-    <div
-      role="button"
+    <Tag
```

Comment:
I think we can add the `disabled` prop here so we can get the correct styles when the multiselect is disabled

Reply:
I was thinking that too and compared it to the current implementation where the tag styling is not changed on disable.

Personally I would also prefer to have the tag disabled since otherwise it really stands out. Do you if there was a reason behind the current implementation, like keeping it well readable for accessibility maybe?

Reply:
it looks like there is a slight color change on the disabled tag svg in the current multiselect, but I'm not sure if it should look like the standalone disabled tag styles cc @laurenmrice

Reply:
Just as a reference: this is how it would look when the tag enters disabled state as well: 
![image](https://user-images.githubusercontent.com/28265588/80707940-13a8d200-8aeb-11ea-9155-b27590eba1b3.png)

Reply:
Let me make a spec for this 👍🏻

Reply:
This is what we can do for a selected-disabled state.

![tags](https://user-images.githubusercontent.com/43969356/80716095-daf20400-8abc-11ea-9470-45f4bb2e685b.png)

Reply:
Thanks for the quick specs, @laurenmrice 👍 
I updated the tag color for the disabled state accordingly, you can check it out in the deploy preview: https://5eaae7cd3ab6150007c0340b--carbon-components-react.netlify.app/?path=/story/multiselect--default